src/styles/components/chat/_chat-prompt-wrapper.scss

```diff
@@ -67,6 +67,8 @@
                     width: 100%;
                     max-height: 20vh;
                     line-height: var(--mynah-line-height);
+                    white-space: pre-wrap;
```

Comment:
This one along with the other `white-space` addition on bottom solves the word breaking during wrapping. Which will match both for text-area and representer span.